tensorboard/functionaltests/wct_test_driver.py

```diff
@@ -0,0 +1,81 @@
+# Copyright 2018 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""WebDriver for running TypeScript and Polymer unit tests."""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import os
+import re
+import subprocess
+import unittest
+from selenium.webdriver.support import expected_conditions
+from selenium.webdriver.support import wait
+from testing.web import webtest
+
+
+# As emitted in the "Listening on:" line of the WebfilesServer output.
+# We extract only the port because the hostname can reroute through corp
+# DNS and force auth, which fails in tests.
+_URL_RE = re.compile(r'http://[^:]*:([0-9]+)/')
+
+
+def create_test_class(binary_path, web_path):
+  """Create a unittest.TestCase class to run WebComponentTester tests.
+
+  Arguments:
+    binary_path: relative path to a `tf_web_library` target containing
+        the tests; e.g.: "tensorboard/components/vz_foo/test/test"
```

Comment:
This should actually be `test/test_web_library` (after a rename). What
do you mean by `test/target`?

Reply:
> `test_web_library` is a name of a build target and, in the most case,
> this will be `test_web_library` if we follow the convention.

Right—so the documentation text should say `test_web_library`, right?
Not `target`?